trade-finance-manager-ui/templates/facilities/facilities.njk

```diff
@@ -27,9 +27,8 @@
       <a
          class="govuk-link"
          href="/case/{{ facility.dealId }}/facility/{{ facility.facilityId }}"
-         aria-label="View facility details"
          data-cy="deal-{{ facility.facilityId }}-ukef-facility-id-link">
-         <span data-cy="deal-{{ facility.facilityId }}-ukef-facility-id-link-text">{{ facility.ukefFacilityId }}</span>
+         <span class="govuk-visually-hidden">View facility </span> <span data-cy="deal-{{ facility.facilityId }}-ukef-facility-id-link-text">{{ facility.ukefFacilityId }}</span> <span class="govuk-visually-hidden">details</span>
```

Comment:
```suggestion
         <span class="govuk-visually-hidden">View facility</span> <span data-cy="deal-{{ facility.facilityId }}-ukef-facility-id-link-text">{{ facility.ukefFacilityId }}</span> <span class="govuk-visually-hidden">details</span>
```

Reply:
You've already got a space between the `<span>`s, so I think you shouldn't have this space.

Reply:
(it looks like this file isn't yet covered by a component test, which would've picked this double space up - Mark has raised this on https://github.com/UK-Export-Finance/dtfs2/pull/2347 and I'm happy for this test to be added as part of another PR)

Reply:
Removed space between the `<span>`